src/unix/android-ifaddrs.c

```diff
@@ -35,6 +35,7 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <netinet/in.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
+#include <linux/if_packet.h>
```

Comment:
Why is this needed?

Reply:
otherwise it does not build (sockaddr_ll definition).